themes/tiddlywiki/vanilla/base.tid

```diff
@@ -2853,19 +2867,22 @@ a.tc-tiddlylink.tc-plugin-info:hover > .tc-plugin-info-chunk .tc-plugin-info-sta
 	color: <<colour foreground>>;
 }
 
-.tc-chosen > .tc-tiddlylink:before {
```

Comment:
IMO we need to keep these configuration for backwards compatibility. We can add a "deprecated" comment, but plugin authors may use these styles. 

I'll need to make some more tests, with the palette-manager edition.

Reply:
@Jermolene -- The tools-buttons: theme, layout, storyview and palette use this CSS. So IMO we can not remove it

Reply:
All the "chosen" indicators fail now.

![image](https://github.com/user-attachments/assets/ff46463d-babd-4067-92ec-0052594dc67d)

Reply:
Thanks @pmario, fixed in c6074402bbdcb8ef04c575090499874d02339a56